src/assets/scripts/client/ui/UiController.js

```diff
@@ -156,6 +156,16 @@ class UiController {
          */
         this.$log = null;
 
+        /**
+         * Duration for chat log transmissions to remain visible
+         *
+         * @for UiController
+         * @property $chatLogDuration
+         * @type number
+         * @default null
+         */
+        this.$chatLogDuration = null;
```

Comment:
you can remove the `$` since this is a numbr. The `$` is reserved only for jquery elements.